src/root/longdouble.c

```diff
@@ -50,7 +50,9 @@ bool initFPU()
     int old_cw = _control87(_MCW_EM | _PC_64  | _RC_NEAR,
                             _MCW_EM | _MCW_PC | _MCW_RC);
 #endif
+#if _MSC_VER < 1700
```

Comment:
The article I found referring to this problem used 1700... this is definitely new to 2015? Wasn't introduced in the version prior?